src/docs/sphinx/quickstart_guide/config_build.rst

```diff
@@ -276,76 +307,151 @@ directly to CMake; for example::
 
 
 CMake Configuration Options
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+---------------------------
+
+The tables in this section summarize the main build system options in Axom.
 
-Here are the key build system options in Axom:
+Axom components, tests, examples, etc.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 +------------------------------+---------+----------------------------------------+
 | OPTION                       | Default | Description                            |
 +==============================+=========+========================================+
-| AXOM_DEBUG_DEFINE            | DEFAULT | Controls whether the `AXOM_DEBUG`      |
-|                              |         | compiler define is enabled             |
-|                              |         |                                        |
-|                              |         | By DEFAULT, it is enabled for          |
-|                              |         | `Debug` and `RelWithDebInfo` configs   |
-|                              |         | but this can be overridden by setting  |
-|                              |         | `AXOM_DEBUG_DEFINE` to `ON` or `OFF`   |
-+------------------------------+---------+----------------------------------------+
 | AXOM_ENABLE_ALL_COMPONENTS   | ON      | Enable all components by default       |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_<FOO>            | ON      | Enables the axom component named 'foo' |
+| AXOM_ENABLE_<FOO>            | ON      | Enable the axom component named 'Foo'  |
 |                              |         |                                        |
 |                              |         | (e.g. AXOM_ENABLE_SIDRE)               |
-|                              |         | for the sidre component                |
+|                              |         | for the Sidre component                |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_DOCS             | ON      | Builds documentation                   |
+| AXOM_ENABLE_EXAMPLES         | ON      | Build Axom examples                    |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_EXAMPLES         | ON      | Builds examples                        |
+| AXOM_ENABLE_TESTS            | ON      | Build Axom unit tests                  |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_TESTS            | ON      | Builds unit tests                      |
+| ENABLE_BENCHMARKS            | OFF     | Enable Google benchmark                |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_TOOLS            | ON      | Builds tools                           |
```

Comment:
The tools were moved to a developer section, but I would argue that they are primarily aimed at axom's users.
E.g. they are the only executables that are installed with the axom libraries as part of `make install`.

My vote is to move that line back to this table.

Reply:
done.